rocAL/rocAL_pybind/examples/parse_config.py

```diff
@@ -79,6 +85,12 @@ def parse_args():
     python_unit_test = parser.add_argument_group('python-unittest', 'python-unittest-related options')
     python_unit_test.add_argument('--augmentation-name', '-aug_name', type=str, default="resize",
                         help='refer python unit test for all augmentation names ')
+    python_unit_test.add_argument('--output-file-name', '-f', type=str, default="",
+                        help='file name to save the augmentation outputs')
+    python_unit_test.add_argument('--interpolation-type', '-i', type=int, default=1,
```

Comment:
Do we need to include interpolation type and scaling modes here ?

Mainly these parse args are used for common arguments for Unit tests.
This looks specific to Resize. Add it as a part of the Unit test if that makes sense.

Reply:
These arguments are needed since we are making the python unittests on par with c++ unit tests. We are passing teh interpolation and scaling modes as arguments there

Reply:
Then name it resize-scaling-mode & resize-interpolation-mode as that would specify these args are to be menat for resize only